script/update-docs.rb

```diff
@@ -443,7 +445,7 @@ def index_doc(filter_tags, doc_list, get_content)
         # HTML anchor on hdlist1 (i.e. command options)
         html.gsub!(/<dt class="hdlist1">(.*?)<\/dt>/) do |_m|
           text = $1.tr("^A-Za-z0-9-", "")
-          anchor = "#{path}-#{text}"
+          anchor = "#{txt_path}-#{text}"
```

Comment:
Maybe we should add a comment, saying that this is needed to preserve backwards-compatibility with already-existing deep links out there on the intertubes.